tests/core/BearTest.py

```diff
@@ -1,14 +1,15 @@
 from itertools import permutations
 from os.path import abspath, exists, isfile, join, getmtime
 import shutil
-from time import sleep
 import unittest
 
+import datetime
 from dependency_management.requirements.PipRequirement import PipRequirement
 
 import requests
 
 import requests_mock
+from freezegun import freeze_time
```

Comment:
be consistent with grouping those new imports ;)
I think there are some possibilities:
1. Put `datetime` into system imports (following alphabetical order) and put freezegun as an own external import between dependency-management and requests (due to alphabetical order):
   ```
   import datetime
   from itertools import permutations
   from os.path import abspath, exists, isfile, join, getmtime
   import shutil
   from time import sleep
   import unittest

   from dependency_management.requirements.PipRequirement import PipRequirement
 
   from freezegun import freeze_time

   import requests
 
   import requests_mock
   ```

2. group `freezegun` and `datetime` together because they are directly related somewhere inside the imports.

Imo 1. is nicer, but up to you. Maybe you find another suiting order ;)